pkg/unstruct/v1alpha2/unstruct.go

```diff
@@ -55,51 +55,51 @@ func Builder() *builder {
 }
 
 // BuilderForYaml returns a new instance of
-// unstruct builder by making use of the provided
+// Unstruct Builder by making use of the provided
 // YAML
-func BuilderForYaml(doc string) *builder {
-	b := Builder()
-	err := yaml.Unmarshal([]byte(doc), &b.unstruct.object)
+func BuilderForYaml(doc string) *Builder {
+	b := NewBuilder()
+	err := yaml.Unmarshal([]byte(doc), &b.Unstruct.object)
 	if err != nil {
 		b.errs = append(b.errs, err)
 	}
 	return b
 }
 
 // BuilderForObject returns a new instance of
-// unstruct builder by making use of the provided object
-func BuilderForObject(obj *unstructured.Unstructured) *builder {
-	b := Builder()
-	b.unstruct.object = obj
+// Unstruct Builder by making use of the provided object
+func BuilderForObject(obj *unstructured.Unstructured) *Builder {
+	b := NewBuilder()
+	b.Unstruct.object = obj
 	return b
 }
 
-// Build returns the unstruct object created by
-// the builder
-func (b *builder) Build() (*unstruct, error) {
+// Build returns the Unstruct object created by
+// the Builder
+func (b *Builder) Build() (*Unstruct, error) {
 	if len(b.errs) != 0 {
 		return nil, errors.Errorf("%v", b.errs)
 	}
-	return b.unstruct, nil
+	return b.Unstruct, nil
 }
 
-// unstructList contains a list of unstructured
+// UnstructList contains a list of Unstructured
 // items
-type unstructList struct {
-	items []*unstruct
+type UnstructList struct {
+	items []*Unstruct
 }
 
-// listBuilder enables building a list
-// of an unstruct instance
-type listBuilder struct {
-	list unstructList
+// ListBuilder enables building a list
+// of an Unstruct instance
+type ListBuilder struct {
+	list UnstructList
 	errs []error
 }
 
 // ListBuilderForYamls returns a mew instance of
-// list unstruct builder by making use of the provided YAMLs
-func ListBuilderForYamls(docs string) *listBuilder {
-	lb := &listBuilder{list: unstructList{}}
+// list Unstruct Builder by making use of the provided YAMLs
+func ListBuilderForYamls(docs string) *ListBuilder {
```

Comment:
Let us make this variadic.
Each string can have one or more yaml definitions.

Reply:
done